vyper/types/convert.py

```diff
@@ -30,23 +30,28 @@
 @signature(('uint256', 'bytes32', 'bytes', 'bool'), '*')
 def to_int128(expr, args, kwargs, context):
     in_node = args[0]
-    typ, len = get_type(in_node)
-    if typ in ('uint256', 'bytes32'):
+    input_type, len = get_type(in_node)
```

Comment:
Just change len to underscore and we can merge ;)

Reply:
@jacqueswww done! 🙌 😅